### PR TITLE
refactor(app): styling updates for app user testing

### LINKED ISSUE #8387: Protocol Upload Revamp: Calibrate CTA and Rewording Pipette Generation Mismatch
### Overview

Tweaked design of the Pipette Offset Calibration row can be viewed [here](https://www.figma.com/file/JCQNOTTK9tTmYPhRqeOfSj/Milestone-0:-Protocol-Upload-Revamp?node-id=3866%3A6353)

### Acceptance Criteria

In the event of a pipette generation mismatch (e.g. protocol requests p300 Single GEN1, robot has p300 Single GEN2):
- [ ] italicized copy should now read "Pipette generation mismatch"
- [ ] modal link text copy should read "Learn more about pipette mismatches"
- [ ] if no pipette offset calibration is present for the attached pipette, present the "CALIBRATE NOW" CTA primary button to the right of the italicized copy and explanation modal link
- [ ] if pipette offset calibration is present for the attached pipette, only present the italicized copy and the explanation modal link.

### PR DESCRIPTION
**Pull request by smb2268**
fix #8387

# Overview

Styling changes for robot cal step

# Changelog

1. Reword pipette mismatch alert and display for both calibrated and attached states
2. Change empty circle next to unfinished calibrations to warning icon
3. Collapse all steps to begin with as opposed to opening robot cal step by default
4. Ask user to confirm recalibration before showing calibrate modal when they click "Recalibrate" for tiplength

# Review requests

- [ ] 1. Reword pipette mismatch alert and display for both calibrated and attached states
- [ ] 2. Change empty circle next to unfinished calibrations to warning icon
- [ ] 3. Collapse all steps to begin with as opposed to opening robot cal step by default
- [ ] 4. Ask user to confirm recalibration before showing calibrate modal when they click "Recalibrate" for tiplength

# Risk assessment

Low

### CODE REVIEW

File: app/src/organisms/ProtocolSetup/RunSetupCard/RobotCalibration/DeckCalibration.tsx [hunk 1]
```diff
@@ -145,7 +145,7 @@ export function DeckCalibration(props: Props): JSX.Element | null {
                 {t('pipette_offset_cal')}
               </Text>
               <Text fontSize={FONT_SIZE_BODY_1} marginTop={SPACING_3}>
-                {t('pipette_offset_cal_description')}
+                <Trans t={t} i18nKey="pipette_offset_cal_description" />
```
Comment (b-cooper) [inline app/src/organisms/ProtocolSetup/RunSetupCard/RobotCalibration/DeckCalibration.tsx:148]: what's the reason for this change? it looks like the t function call should have been sufficient?

**Review by b-cooper — APPROVED** [review]
Everything looks good to me. 🍍

**Review by jerader — APPROVED** [review]
lgtm 🚀 it looks so good with the style changes!

File: app/src/organisms/ProtocolSetup/RunSetupCard/RobotCalibration/DeckCalibration.tsx (hunk 1 above)
Comment (smb2268) [reply app/src/organisms/ProtocolSetup/RunSetupCard/RobotCalibration/DeckCalibration.tsx:148]: There are a couple of `<br/>` tags in this that show up as plain text if it isn't in a `Trans` component.
